### ISSUE TITLE
Buttons added in the CMS are missing aria-label

### ISSUE THREAD

**Issue by greensomes** [DOI-ONRR/onrr.gov-site#2598]

**Comment by greensomes** [comment]
Actually, it wasn't that the links needed aria-label. The issue is that some links added in the CMS add what are more or less "phantom" links - the links is duplicated and is not visible and focusable. Accessibility rules don't like this. I removed the "phantom" link from the card in the CMS and the score is now 100.
